.github/workflows/bunActions.yml

```diff
@@ -0,0 +1,23 @@
+name: Bun actions
+on:
+  pull_request:
+    branches: ["develop", "main"]
+  push:
+    branches: ["develop", "main", "**action**"]
+jobs:
+  build:
+    name: Build backend with bun
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: oven-sh/setup-bun@v2
+        with:
+          bun-version: 1.1.12
```

Comment:
Is that the correct version? It does not match the types-version?

Reply:
Yes, it is the version from the test system and the one I used for development.